CHANGELOG.md

```diff
@@ -1,5 +1,14 @@
 # Changelog
 
+### 4.4.6
+
+This release of teleport contains a security fix and a bug fix.
+
+* Patch a SAML authentication bypass (see https://github.com/russellhaering/gosaml2/security/advisories/GHSA-xhqq-x44f-9fgg): [#5120](https://github.com/gravitational/teleport/pull/5120). Users with SAML SSO providers should upgrade as soon as possible.
```

Comment:
I think we should add similar wording to the last time we did one of these patch releases:

```suggestion
* Patch a SAML authentication bypass (see https://github.com/russellhaering/gosaml2/security/advisories/GHSA-xhqq-x44f-9fgg): [#5120](https://github.com/gravitational/teleport/pull/5120).

Any Enterprise SSO users using Okta, Active Directory, OneLogin or custom SAML connectors should upgrade their auth servers to version 4.4.6 and restart Teleport. If you are unable to upgrade immediately, we suggest disabling SAML connectors for all clusters until the updates can be applied.
```